src/stories/hooks/UseGroups/UseGroupsReturn.md

```diff
@@ -0,0 +1,16 @@
+| property           | type                                | description                                                                                                                 |
+| ------------------ | ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| groups             | `Array<PlpItemGroup>`               | List of transformed groups returned in the API                                                                              |
+| groupOptions       | `Array<PlpItemGroup>`               | List of transformed child groups that can be used as options                                                                |
+| optionsToRender    | `boolean`                           | Dynamic list of child groups to be rendered as options                                                                      |
```

Comment:
`groups` we could probably omit since it's a prop

`groupOptions` is the full list of options whereas `optionsToRender` takes into account the `initialNumOptions` prop and slices the excess out. It seems quite bulky since it's a parent hook that calls a bunch of subhooks, but I think it makes sense since this hook should allow user to not have to call other small hooks. My basis is the other major hooks we've done for the other OS libs like useAutocomplete or useQuizzes but maybe I've understood the pattern wrong.